Makefile-az.mk

```diff
@@ -25,6 +25,32 @@ az-all-custom-vnet: az-login az-create-workload-msi az-mkaks-custom-vnet az-crea
 
 az-all-savm:    az-login az-mkaks-savm az-perm-savm az-patch-skaffold-azure az-build az-run az-run-sample ## Provision the infra (ACR,AKS); build and deploy Karpenter; deploy sample Provisioner and workload - StandaloneVirtualMachines
 
+install-tools: az-tool ## install tools to create a local developer environment
+	sudo apt update && sudo apt install jq -y
+	curl -Lo yq https://github.com/mikefarah/yq/releases/download/v4.43.1/yq_linux_amd64 && chmod +x yq && sudo mv yq /usr/local/bin
+	if test -z "$(shell which helm)"; then \
+		curl -L https://get.helm.sh/helm-v3.14.3-linux-amd64.tar.gz | sudo tar xvfz - linux-amd64/helm && sudo mv linux-amd64/helm /usr/local/bin/helm && rm -rf linux-amd64; \
+	fi
+	if test -z "$(shell which docker)"; then \
+		curl -fsSL https://get.docker.com | bash -; \
+	fi
+	if test -z "$(shell which kubectl)"; then \
+		curl -LO "https://dl.k8s.io/release/$(shell curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" && chmod +x kubectl && sudo mv kubectl /usr/local/bin; \
+	fi
+	if test -z "$(shell which go)" || [ $(shell go version | awk -F'[ .]' '{print$$3"."$$4}') = $(shell grep -E '^go' go.mod | sed 's/ //g') ]; then \
```

Comment:
Oh my mistake.
I will change the "!=" instead of "=". thanks

And the reason why i added $$ instead of $ is explained below link. 
https://stackoverflow.com/questions/15071439/how-do-i-use-pipes-in-a-makefile-shell-command
it would get the valid result (e.g. go.1.2x) when you run `"go version | awk -F'[ .]' '{print$3"."$4}}'` on Makefile.

Reply:
Ok, I see bash within Makefile. Tested there and makes sense. Thanks for sharing over the reference! :)